docs/001_develop/03_client-capabilities/008_interaction/text-area.mdx

```diff
@@ -0,0 +1,163 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Text Area
+
+Implementation of a [text-area](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) component.
+
+Use cases:
+* Comments
+* Descriptions
+* Multi-line text inputs
+* Complex searches
+
+## Example
+
+<TextAreaDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+  <TabItem value="genesis">
+
+    Declaration
+    ```html
+    <rapid-text-area placeholder="Placeholder text"></rapid-text-area>
+    ```
+
+    Usage
+    ```typescript
+    @customElement({
+    name: 'my-element',
+    template: html`
+        <rapid-text-area :value=${sync((x) => x.textAreaValue)}>
+          Sample Text Area
+        </rapid-text-area>
+      `,
+  })
+    export class MyElement extends GenesisElement {
+      @observable textAreaValue = '';
+      textAreaValueChanged() {
+         console.log(this.textAreaValue);
+      }
+    }
+    ```
+
+  </TabItem>
+  <TabItem value="react">
+
+  Declaration
+  ```html
+  <rapid-text-area></rapid-text-area>
+  ```
+  Usage
+  ```typescript
+  export function MyComponent() {
+    const handleTextAreaChanged = (event) => {
+      console.log({ event.target.value })
+    };
+    return (
+      <rapid-text-area onChange={handleTextAreaChanged}></rapid-text-area>
+    );
+  }
+  ```
+</TabItem>
+  <TabItem value="angular">
+
+  Declaration
+  ```html
+  <rapid-text-area></rapid-text-area>
+  ```
+
+  Usage
+  ```typescript
+  import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+  import { FormsModule } from '@angular/forms';
+  @Component({
+    selector: 'my-root',
+    template: `
+          <rapid-text-area
+            (change)="onTextAreaChange($event)"
+          ></rapid-text-area>
+        `,
+    styleUrls: ['./my.component.css'],
+    standalone: true,
+    schemas: [CUSTOM_ELEMENTS_SCHEMA],
+    imports: [FormsModule],
+  })
+  export class AppComponent {
+    onTextAreaChange(event: Event) {
+      const target = event.target as HTMLInputElement;
+      console.log(target.value);
+    }
+  }
+  ```
+</TabItem>
+</Tabs>
+
+## API
+
+Attributes:
+
+| Name        | Type      | Description                                                                                                 |
+|-------------|-----------|-------------------------------------------------------------------------------------------------------------|
+| appearance  | `string`  | Changes the general view of the element; this can be **filled** or **outline**                              |
+| autofocus   | `boolean` | When true, the component will be focused when the page has finished loading                                 |
+| cols        | `boolean` | Defines the size of the element horizontally. **Default: 20**                                               |
```

Comment:
is this mean to be number (as a string)?

Reply:
If referring to cols, yeah should be number, have updated.